pageserver/src/tenant.rs

```diff
@@ -595,10 +595,9 @@ impl Tenant {
         mode: SpawnMode,
         ctx: &RequestContext,
     ) -> anyhow::Result<Arc<Tenant>> {
-        // TODO(sharding): make WalRedoManager shard-aware
```

Comment:
Why do we need to make weal redo shard-aware?
It seems to me that walredo should not just be called for pages not belonging to the shard..

Reply:
See description:
> Make walredo code shard aware, so that its logging includes shard IDs (functionally no change)